### DISCUSSION TITLE
Autorisés les combats amicaux même sur l'île

### DISCUSSION THREAD

**Discussion by Sousoulman** [Crownicles/Crownicles#2344] [Nouvelles idées]
J'ai voulu faire un combat amicaux avec quelqu'un pour sa quête mais vu que j'étais sur l'île je n'ai pas pus alors je me dit que ce serait ce serait bien de pouvoir le faire.

**Comment by Electek** [comment]
oui sa serais chouette si t'attend que le reste des membres de ta guildes arrive que tu puisse te battre pour faire passer le temps.
